Riot/Modules/LocationSharing/LocationManager.swift

```diff
@@ -24,7 +25,7 @@ class LocationManager: NSObject {
     // MARK: - Constants
     
     private enum Constants {
-        static let distanceFiler: CLLocationDistance = 200.0
+        static let distanceFiler: CLLocationDistance = 10.0
```

Comment:
maybe take the opportunity to fix the typo ? :) EDIT: too late, the PR has been merged, ignore me!